slider/index.js

```diff
@@ -0,0 +1,170 @@
+class IRNMNSlider extends HTMLElement {
+
+    CLASSNAMES = [];
+
+    constructor() {
+        super();
+        this.CLASSNAMES = this.selectors;
+        this.currentSlide = 1; // Start from the first visible slide after cloning        
+    }
+
+    /**
+     * Get the selectors from the attribute
+     * to be used as classnames obj
+     * 
+     * @returns {object} - The selectors object
+     */
+    get selectors() {
+        let selectors = this.getAttribute('selectors');
```

Comment:
@samuelsilvapt Feels like a try/ catch could offer a bit of error handling in case no json or correct format.

Reply:
I added try/catch in everything I found necessary. This case, also number of sliders